src/platform/common.h

```diff
@@ -652,9 +652,10 @@ namespace platf {
    * @param input The client-specific input context.
    * @param touch_port The current viewport for translating to screen coordinates.
    * @param touch The touch event.
+   * @param input_ctx The global input context.
    */
   void
-  touch(client_input_t *input, const touch_port_t &touch_port, const touch_input_t &touch);
+  touch(client_input_t *input, const touch_port_t &touch_port, const touch_input_t &touch, input_t &input_ctx);
```

Comment:
If you need the global input context, you should store it inside your `client_input_t` like the Windows implmentation does

Reply:
Done